tagstore/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Convenience module to make importing Tagstore easier.
+"""
+from .tagstore import Tagstore
```

Comment:
Would like to see a README in this new `tagstore` dir too.

Reply:
Sure, `README` added in 1463db8ef1cb9b41d1a64c082782f8119a1e0eb9